### PR TITLE
fix: add a message when flow is down

### PR DESCRIPTION
**Pull request by jupyterjazz**
Closes https://github.com/jina-ai/now/issues/753

Screenshot 2022-12-23 at 11 17 16

---

- [ ] This PR references an open issue
- [ ] Test added
- [ ] Documentation adjusted

### CODE REVIEW

**Comment by jupyterjazz** [comment]
Needs https://github.com/jina-ai/now/pull/813

File: deployment/bff/app/v1/routers/helper.py [hunk 1]
```diff
@@ -124,12 +124,22 @@ def jina_client_post(
...
                 status_code=status.HTTP_401_UNAUTHORIZED,
                 detail='You are not authorised to use this flow',
             )
         else:
-            raise e
-    return result
+            return e
+    elif isinstance(e, BadServerFlow):
+        if 'no route matched' in e.args[0].lower():
+            return Exception('Your flow is down')
```
Comment (joschkabraun) [inline deployment/bff/app/v1/routers/helper.py:143]: Let's rephrase this to something more professional, e.g. `Flow with ID ABC can't be found`

**Review by joschkabraun — CHANGES_REQUESTED** [review]

File: deployment/bff/app/v1/routers/helper.py (hunk 1 above)
Comment (jupyterjazz) [reply deployment/bff/app/v1/routers/helper.py]: good point, what about `flow with host {host} can not be found`?

Comment (joschkabraun) [reply deployment/bff/app/v1/routers/helper.py]: I think the user thinks more in terms of ID (what you get from `jc list`).

Comment (jupyterjazz) [reply deployment/bff/app/v1/routers/helper.py]: makes sense. should I extract it from the host name or is there an easier way to get it?

Comment (jupyterjazz) [reply deployment/bff/app/v1/routers/helper.py]: Done. you can see the screenshot in the description

**Review by azayz — APPROVED** [review]
LGTM

**Review by joschkabraun — APPROVED** [review]
Nice! LGTM
